plugins/auth/token.js

```diff
@@ -16,7 +16,7 @@ module.exports = () => ({
         notes: 'Generate a JWT for use throughout Screwdriver',
         tags: ['api', 'auth', 'token'],
         auth: {
-            strategies: ['token', 'session', 'auth_token'],
+            strategies: ['session', 'auth_token'],
```

Comment:
Talked to @petey about this one, if `'token'` is listed among the strategies for this route then a JWT can be used to "refresh" itself by calling `GET /auth/token`.